Yoast/YoastSEO.js
# Use morphology in devtool

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

~~Branched from `examples-webpack-work`. Only commit of this PR is 0746cd9d54bb92cfcd81e5bd531db3973747e8ff~~

* Added use morphology toggle. Which uses local data.
* Added options reducer and moved `isRelatedKeyphrase` there too. Meant for the data that should not be sent to the worker.
* Fixed the analyze control not respecting the related keyphrase option.

## Test instructions

This PR can be tested by following these steps:

* Go to `examples/webpack`.
* Install packages: `yarn`.
* Run the dev server: `yarn start`.
* Ensure the morphology toggle works. For example check out the difference in the keyphrase density when using different word forms of the keyphrase in the text.
* Ensure related keyphrase toggle still functions.
* Ensure the analyze control runs analyze or an analyze related keywords depending on the is related keyphrase state.

Requires #1953 
Fixes #1961
